### PR TITLE
Update group-mentee-guide

### PR DESCRIPTION
**Pull request by parispittman**
Fixed bullet list for expectations, added SIG list, other formatting

### CODE REVIEW

**Comment by tpepper** [comment]
That fixes the error I saw.

File: mentoring/group-mentee-guide.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ During the three month period, you will work on a path to the next level of memb
...
-## Expectations  
-You understand the requirements to become $member_role (ex: member->reviewer) 
-Good standing throughout the program and uphold our code of conduct
-Attend two bi-weekly standups a month
-The program is at your pace but these meetings will discuss your wins and blocks
-These meetings will also cover development area topics, suggested activities, and groomed issues
-Be helpful to your peers in the cohort
-Give back and be a mentor in a future cohort
-Mutual respect from mentors and peers who are in the cohort with you
+## Expectations
```
Comment (stp-ip) [inline mentoring/group-mentee-guide.md]: whitespace

Comment (parispittman) [reply mentoring/group-mentee-guide.md]: I'm not seeing the whitespace issue - is it between two words?

Comment (stp-ip) [reply mentoring/group-mentee-guide.md]: After "## Expectations" I see 4 spaces. Felt unnecessary, but I might be wrong.

**Pull request merged**